src/Composer/Filter/PlatformRequirementFilter/PlatformRequirementFilterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Composer\Filter\PlatformRequirementFilter;
+
+interface PlatformRequirementFilterInterface
+{
+    /**
+     * @param string $req
+     * @return bool
+     */
+    public function isReqIgnored($req);
```

Comment:
If you don't mind, I think this would be better as simply isIgnored?

```suggestion
    public function isIgnored($req);
```

Reply:
Definitely better, `req` is not needed in this context